### DISCUSSION TITLE
Why the Stylemanager % units not working

### DISCUSSION THREAD

**Discussion by aharishsundhar** [GrapesJS/grapesjs#5166] [Q&A]
@artf If try Height add % unit not working from latest version 0.21.1
![Screenshot from 2023-06-06 15-36-24](https://github.com/GrapesJS/grapesjs/assets/72383148/550ccc5d-a281-48fe-a584-99fc1aef70c6)

**Comment by artf** [comment]
https://stackoverflow.com/questions/5657964/css-why-doesn-t-percentage-height-work
